src/libraries/System.Private.CoreLib/src/System/String.Comparison.cs

```diff
@@ -1162,6 +1186,29 @@ public bool StartsWith(char value)
             return Length != 0 && _firstChar == value;
         }
 
+        public bool StartsWith(char value, StringComparison comparisonType)
+        {
```

Comment:
Same comments I added on `EndsWith` apply here too.

Reply:
This need to be implemented as `public bool StartsWith(string value, StringComparison comparisonType)`